woocommerce/woocommerce
# Fix JS lint errors

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes the lint errors via `pnpm -- turbo run lint:fix --filter='@woocommerce/admin-library...' --filter='!@woocommerce/e2e*' --filter='!@woocommerce/api'`

It looks like somehow eslint config has been changed, so we got a bunch of new lint errors.

### How to test the changes in this Pull Request:

1. Run `pnpm -- turbo run lint --filter='@woocommerce/admin-library...' --filter='!@woocommerce/e2e*' --filter='!@woocommerce/api'` in trunk -> many lint errors
2. Run `pnpm -- turbo run lint --filter='@woocommerce/admin-library...' --filter='!@woocommerce/e2e*' --filter='!@woocommerce/api'`  in this branch  without such errors
3. Lint JS CI should pass https://github.com/woocommerce/woocommerce/runs/6934596189?check_suite_focus=true

Feel free to approve and merge this PR. 

no changelog

### Other information:




### FOR PR REVIEWER ONLY:
